picoquic/sockloop.c

```diff
@@ -764,6 +770,9 @@ void* picoquic_packet_loop_v3(void* v_ctx)
                 }
             }
         }
+        else {
+            nb_loop_immediate++;
+        }
         loop_immediate = 0;
```

Comment:
I wonder aren't we resetting loop_immediate to 0 always here ?

Reply:
Yes, it will be set to 1 later if there is a reason.